__tests__/reducers/index.test.js

```diff
@@ -130,20 +132,88 @@ describe(`Takes a resource template ID and populates the global state`, () => {
   })
 
   it('tests with a more realistic payload with defaults', () => {
+    const testId = jest.spyOn(shortid, 'generate').mockReturnValue(0)
     const defaultStateResult = refreshResourceTemplate({}, {
       type: 'REFRESH_RESOURCE_TEMPLATE',
       payload: {
         reduxPath: ['resourceTemplate:bf2:Item', 'http://schema.org/name'],
-        defaults: ['Sinopia Name']
+        property: { valueConstraint: { defaults: [ { defaultLiteral: 'Sinopia Name' } ] }}
       }
     })
     expect(defaultStateResult).toEqual({
       'resourceTemplate:bf2:Item': {
         'http://schema.org/name':{
-          items: [ "Sinopia Name" ]
+          items: [ {content: "Sinopia Name", id: 0, uri: undefined } ]
         }
       }
     })
   })
 
 })
+
+describe('Takes a property and returns an empty or a populated array', () => {
```

Comment:
I think i would change this to ... `populateDefaults() method` as the tests enclosed seem to be testing that ...

Reply:
Now changed.